js/google_ima.js

```diff
@@ -1623,6 +1703,40 @@ require("../html5-common/js/utils/utils.js");
             }
 
             break;
+          case eventType.IMPRESSION:
+
+            this.currentImpressionTime = new Date().valueOf();
+            var loadTime = this.currentImpressionTime - this.adResponseTime;
```

Comment:
Did we want to track from ad request or ad response? Just double checking.

Reply:
The Analytics Events MVP doc defines load time to be: time between Ad Request Success and STARTED/first frame. I took Ad Request Success to mean when that the ad response is retrieved without error, which is what this.adResponseTime stores.